build/uninstall-privileged-service.ps1

```diff
@@ -0,0 +1,16 @@
+# .SYNOPSIS
+# PowerShell script to uninstall Rancher Desktop Privileged Service
+
+param($InstallDir)
+
+# .SYNOPSIS
+# Uninstalls Rancher Desktop Privileged Service
+function Uninstall-PrivilegedService {
+  Param([String] $InstallDir)
+  Write-Output "Uninstalling Rancher Desktop Privileged Service"
+  $InstallCommand = "`"${InstallDir}`" uninstall"
+  Invoke-Expression -Command $InstallCommand
```

Comment:
```suggestion
  & ${InstallDir} uninstall
```
Though presumably it should be called `$ServiceExecutable` or something along those lines instead.